storage/scylla_importer.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include <zmq.h>
+#include "accelerator/config.h"
+#include "cclient/service.h"
+#include "scylla_api.h"
+
+#define SCYLLA_IMPORTER_LOGGER "SCYLLA_IMPORTER"
+
+static logger_id_t logger_id;
+void scylla_importer_logger_init() { logger_id = logger_helper_enable(SCYLLA_IMPORTER_LOGGER, LOGGER_DEBUG, true); }
+
+int scylla_importer_logger_release() {
+  logger_helper_release(logger_id);
+  if (logger_helper_destroy() != RC_OK) {
+    log_critical(logger_id, "%s.\n", SCYLLA_IMPORTER_LOGGER);
+    return EXIT_FAILURE;
+  }
+  return 0;
+}
+
+static status_t import_historical_data(CassSession* session, char* file_path) {
+  /* The format of each line in dump files is HASH,MessageAddressTagBundleTrunkBranchNonce */
+#define BUFFER_SIZE                                                                                  \
```

Comment:
How about using `const uint32_t` instead of `define`. If we use const to delcare a scalar value the compilier may treat it as an expanding constant value.